sharded-redis-client.js

```diff
@@ -180,12 +180,19 @@ shardable.forEach(function(cmd){
     var startIndex = client._rrindex;
     var commandFn = client[cmd];
     var wrappedClient = self._getWrappedClient(key);
+    var logger = self._logger;
 
     var mainCb = args[args.length - 1];
     if(typeof mainCb !== "function") mainCb = args[args.length] = noop;
 
     args[args.length - 1] = function (err) {
-      if (err) console.error(new Date().toISOString(), 'sharded-redis-client [' + client.address + '] err: ' + err);
+        if (logger && logger.warn) {
+          logger.warn(new Date().toISOString(), 'sharded-redis-client [' + client.address + '] err: ' + err);
```

Comment:
I don't think this is the bunyan interface. The first argument should be an object that the error is in, e.g. `{err: err, clientAddress: client.address}` and the date is provided automatically.